src/models/Models.jl

```diff
@@ -10,8 +10,11 @@ using ImageTransformations
 using MLDataPattern
 using ProgressMeter: Progress, next!
 
+include("utils/utils.jl")
```

Comment:
Does this need to be in a subdirectory? I think it only makes sense if we add more similar files there.

Reply:
It is not needed but i did it because Honza made this directory in his branch/pull request. So i thought i can just use it.

Reply:
yeah then lets keep it